src/rpc/rawtransaction.cpp

```diff
@@ -1598,9 +1606,11 @@ static RPCHelpMan utxoupdatepsbt()
     CCoinsView viewDummy;
     CCoinsViewCache view(&viewDummy);
     {
-        const CTxMemPool& mempool = EnsureMemPool(request.context);
+        NodeContext& node = EnsureAnyNodeContext(request.context);
+        const CTxMemPool& mempool = EnsureMemPool(node);
+        ChainstateManager& chainman = EnsureChainman(node);
```

Comment:
I haven't commented on all of these examples, but I really do think readability is helped by placing the `Ensure*()` assumptions at the top of the rpc methods.